pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1433,6 +1433,12 @@ type GatewaySpec struct {
 	// +optional
 	Placement Placement `json:"placement,omitempty"`
 
+	// Whether to disable the Multisite synchronization traffic for the rgw daemon. If not set, the
+	// multisite synchronization traffic will be enabled.
+	// +nullable
+	// +optional
+	DisableMultisiteSyncTraffic bool `json:"disableMultisiteSyncTraffic,omitempty"`
```

Comment:
Per Casey and Ali's comments, let's rework this a little to be extra clear for users that this only affects the send/transmit portion of multisite sync.

```suggestion
	// Prevent this object store's gateways from transmitting multisite replication data. Note that this this value does not affect whether gateways receive multisite replication traffic. If false or unset, this object store's gateways will be able to transmit multisite replication data. 
	// +optional
	DisableMultisiteSyncTraffic bool `json:"disableMultisiteSyncTraffic,omitempty"`
```